class.jetpack.php

```diff
@@ -5363,8 +5363,20 @@ function allow_wpcom_public_api_domain( $domains ) {
 		return $domains;
 	}
 
+	static function is_redirect_encoded( $redirect_url ) {
+		return wp_startswith( $redirect_url, 'https%3A%2F%2F' );
```

Comment:
This only catches https urls -- would it be wiser to catch `/https?/i` starting (plus the encoded slashes)? -- some could capitalize the protocol (unlikely, but possible and may as well hit the edge cases)